src/bp-document/bp-document-filters.php

```diff
@@ -1937,3 +1939,29 @@ function bb_setup_template_for_document_preview( $template ) {
 
 	return $template;
 }
+
+/**
+ * Function will check content empty or not for the document.
+ * If content will empty then return true and allow empty content in DB for the document.
+ *
+ * @param bool         $validated_content Boolean from filter.
+ * @param string       $content           Message content.
+ * @param array|object $id                Message ID.
+ *
+ * @return bool
+ */
+function bb_check_is_get_message_content_validate_with_document( $validated_content, $content = '', $message_id ) {
+	if ( ! empty( $content ) ) {
```

Comment:
@milanhirapra Why we need to check empty content again it's already checked before